test/test-enjoi.js

```diff
@@ -119,9 +119,9 @@ Test('enjoi extensions', function (t) {
 
         const enjoi = Enjoi.defaults({
             extensions: [
-                {
```

Comment:
Can we also keep a test for object since the extensionSchema includes such an option?

Reply:
this case is still tested above on lines 79 & 96